UT4MasterServer.Web/src/pages/Admin/Accounts/Accounts.vue

```diff
@@ -75,4 +90,29 @@ async function loadAccounts() {
     }
 }
 
+function canDelete(account: IGridAccount) {
+    return AccountStore.account?.ID !== account.ID && !account.Roles?.includes(AccountFlag.Admin);
+}

```

Comment:
Similar as for clients. I don't think its really up to the frontend to check whether the operation is possible. Backend might change and then frontend will still use old conditions. Which means that both frontend and backend need to be changed if any conditions might need to change.

I can already tell you that this condition is not equivalent to the condition that backend checks. Moderators are also capable of deleting accounts and no one is capable of deleting account on the same or higher level of status (in order: admin, moderator, others).